.github/workflows/main.yaml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    container:
+      image: google/dart:latest
```

Comment:
I would prefer to use Flutter stable and the corresponding Dart version

Reply:
Well for dart_test and angular_bloc you shouldn't use Flutter SDK, ay? So for that, I have included Dart also.

Reply:
yeah I agree that we need the Dart SDK but would like to use whatever version of Dart the latest stable version of Flutter is using.

Reply:
Hmm. Could we use flutter too for that? `flutter pub get` etc should work in angular and bloc_test as well?